website/docs/reference/playground.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: The Playground
+---
+
+:::info Availability
+
+The Unleash playground is available from Unleash 4.14 onwards. The playground is available in all Unleash versions.
+
+:::
+
+![The Unleash Playground form and an indication of where in the nav menu it is located.](/img/playground-form.png)
+
+The unleash playground is a part of the Unleash Admin UI and an accompanying API. The playground lets you see which of your instance's feature toggles would be enabled for a given Unleash context. It has multiple uses, such as:
+
+-   Understanding how Unleash evaluates strategies, constraints, and segments.
+-   Debugging Unleash contexts and toggles behaving differently from what you expect.
+-   Tailoring a set of strategies for given contexts.
+
+Each feature toggle will contain info on whether it was enabled or not and which variant it was assigned. Later versions of the playground also contain [detailed evaluation results](#detailed-evaluation-results) to help you understand exactly why the feature was enabled or disabled.
+
+## Configuring the query
+
+This section describes what information the playground needs to evaluate your Unleash context against your existing features.
+
+### Environment and projects
+
+In addition the actual Unleash context, the playground also needs to know about environments and projects. In addition to the Unleash context, you must also select an environment and one or more projects for the query.
```

Comment:
```suggestion
By default first environment on the list is selected, and feature toggles for all projects in this environment will be evaluated.
```

Reply:
Reworded in ab1b25b9c6da734f6a1bc20c6627228a75ef6058 to be more in line with your suggestion. What do you think?